src/extension.ts

```diff
@@ -59,33 +59,34 @@ async function initClient(serverInfo: server.ServerInfo): Promise<RSPClient> {
         vscode.commands.registerCommand('server.debug',
-            context => executeCommand(commandHandler.debugServer, commandHandler, context)),
+            context => executeCommand(commandHandler.debugServer, commandHandler, context, errorMessage.replace('%ACTION%', 'debug'))),
         vscode.commands.registerCommand('server.stop',
-            context => executeCommand(commandHandler.stopServer, commandHandler, context)),
+            context => executeCommand(commandHandler.stopServer, commandHandler, context, errorMessage.replace('%ACTION%', 'stop'))),
         vscode.commands.registerCommand('server.remove',
-            context => executeCommand(commandHandler.removeServer, commandHandler, context)),
+            context => executeCommand(commandHandler.removeServer, commandHandler, context, errorMessage.replace('%ACTION%', 'remove'))),
         vscode.commands.registerCommand('server.output',
-            context => executeCommand(commandHandler.showServerOutput, commandHandler, context)),
+            context => executeCommand(commandHandler.showServerOutput, commandHandler, context, '')),
         vscode.commands.registerCommand('server.addDeployment',
-            context => executeCommand(commandHandler.addDeployment, commandHandler, context)),
+            context => executeCommand(commandHandler.addDeployment, commandHandler, context, errorMessage.replace('%ACTION%', 'add deployment to'))),
         vscode.commands.registerCommand('server.removeDeployment',
-            context => executeCommand(commandHandler.removeDeployment, commandHandler, context)),
+            context => executeCommand(commandHandler.removeDeployment, commandHandler, context, errorMessage.replace('%ACTION%', 'remove deployment to'))),
         vscode.commands.registerCommand('server.publishFull',
-            context => executeCommand(commandHandler.fullPublishServer, commandHandler, context)),
+            context => executeCommand(commandHandler.fullPublishServer, commandHandler, context, errorMessage.replace('%ACTION%', 'publish to'))),
         vscode.commands.registerCommand('server.createServer',
-            () => executeCommand(commandHandler.createServer, commandHandler)),
+            () => executeCommand(commandHandler.createServer, commandHandler, errorMessage.replace('%ACTION%', 'create'))),
         vscode.commands.registerCommand('server.addLocation',
-            () => executeCommand(commandHandler.addLocation, commandHandler)),
+            () => executeCommand(commandHandler.addLocation, commandHandler, 'Unable to detect any server: ')),
         vscode.commands.registerCommand('server.downloadRuntime',
-            () => executeCommand(commandHandler.downloadRuntime, commandHandler)),
+            () => executeCommand(commandHandler.downloadRuntime, commandHandler, 'Unable to download the runtime: ')),
```

Comment:
I think that we should always talk of server to the user to be consistent.
Alternatively we should also talk of a runtime in the case right above ```Unable to detect any runtime:```